samples/react-native/android/app/src/main/jni/CMakeLists.txt

```diff
@@ -29,7 +29,3 @@ project(appmodules)
 
 # This file includes all the necessary to let you build your application with the New Architecture.
 include(${REACT_ANDROID_DIR}/cmake-utils/ReactNative-application.cmake)
-
-# App needs to add and link against tm (TurboModules) folder
-add_subdirectory(${REACT_ANDROID_DIR}/../../../tm/ tm_build)
-target_link_libraries(${CMAKE_PROJECT_NAME} tm)
```

Comment:
Q: This was unused or breaking on RN 0.75?

Reply:
Good catch @lucas-zimerman 👍  I was facing a build issue (`.../jni/OnLoad.cpp:35:10: fatal error: 'rncli.h' file not found #include <rncli.h>`) on RN 0.75 and I fixed it by picking the [CMakeLists.txt](https://github.com/facebook/react-native/blob/v0.75.4/packages/react-native/ReactAndroid/cmake-utils/default-app-setup/CMakeLists.txt) and [OnLoad.cpp](https://github.com/facebook/react-native/blob/v0.75.4/packages/react-native/ReactAndroid/cmake-utils/default-app-setup/OnLoad.cpp) from the RN 0.75.4 sample app.

Comparing with the [0.73.9 CMakeLists.txt file](https://github.com/facebook/react-native/blob/v0.73.9/packages/react-native/ReactAndroid/cmake-utils/default-app-setup/CMakeLists.txt) I realised that [those are our own additions](https://github.com/getsentry/sentry-react-native/pull/2522/files#diff-14ee27ec177f3b1084534f54a4f613295f869448f42d0863a715e8eb26fcc101) so I reverted the change with 3b165dc4cc8cfe796619cef5ba0f8d360b381086